releases/release-1.20/README.md

```diff
@@ -71,7 +71,7 @@ The 1.20 release cycle is proposed as follows:
 | **KubeCon CloudNativeCon North America Ends** | Community | Friday, November 20 |  | |
 | Docs deadline - PRs ready for review | Docs Lead | Mon November 23 | week 11 | |
 | **[Test Freeze]** (EOD PST) | Branch Manager | Mon November 23 | |
-| Docs complete - All PRs reviewed and ready to merge | Docs Lead | Mon November 30 | week 12 | |
+| Docs complete - All PRs reviewed and ready to merge | Docs Lead | Wed December 2 | week 12 | |
```

Comment:
There is a `TL;DR` section above that also needs to reflect the new date :)

Reply:
Doh!!

Thank you :)